src/spikeinterface/postprocessing/spike_locations.py

```diff
@@ -95,12 +116,16 @@ def get_extension_function():
 
 WaveformExtractor.register_extension(SpikeLocationsCalculator)
 
+# @alessio @pierre: channel_from_template=True is the old behavior but this is not accurate
+# what do we put by default ?
+
```

Comment:
I would go for the new behavior as a default, but we need to think on the impact with the metrics